### DISCUSSION TITLE
Llama-2-GGML-Medical-Chatbot

### DISCUSSION THREAD

**Discussion by ThisIs-Developer** [community/community#114226] [Other Feature Feedback, Questions, & Ideas]
### Llama-2-GGML-Medical-Chatbot

Product Feedback

### Body

How can it **streamline** communication and improve patient care?
repo: [Llama-2-GGML-Medical-Chatbot](https://github.com/ThisIs-Developer/Llama-2-GGML-Medical-Chatbot)
Huggingface: https://huggingface.co/spaces/ThisIs-Developer/Llama-2-GGML-Medical-Chatbot

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ThisIs-Developer ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

If you're asking how to share feedback or ask a question about a piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
